forms-flow-web/src/apiManager/endpoints/index.js

```diff
@@ -34,6 +34,7 @@ const API = {
   PUBLISH: `${WEB_BASE_URL}/form/<mapper_id>/publish`,
   UN_PUBLISH: `${WEB_BASE_URL}/form/<mapper_id>/unpublish`,
   FORM_HISTORY: `${WEB_BASE_URL}/form/form-history`,
+  PROCESS_HISTORY: `${WEB_BASE_URL}/process`,
```

Comment:
we dont need this line.instead use GET_PROCESSES_DETAILS in the components where u used PROCESS_HISTORY

GET_PROCESSES_DETAILS is already declared in line 28
@Ajay-aot

Reply:
updated